content/docs/command-reference/gc.md

```diff
@@ -145,3 +164,6 @@ Let's check the size now:
 $ du -sh .dvc/cache/
 3.1G    .dvc/cache/
 ```
+
+[cache is shared]: /doc/user-guide/how-to/share-a-dvc-cache
+[dvc experiments]: /doc/user-guide/experiment-management#experiments
```

Comment:
💅🏼 Ideally we should place these relinks at the end of the **Options** section (before the **Examples** H2), where they're used.